modyn/tests/storage/internal/file_wrapper/binary_file_wrapper_test.cpp

```diff
@@ -48,6 +49,7 @@ class BinaryFileWrapperTest : public ::testing::Test {
 TEST_F(BinaryFileWrapperTest, TestGetNumberOfSamples) {
   const std::shared_ptr<std::ifstream> stream_ptr = std::make_shared<std::ifstream>();
   stream_ptr->open(file_name_, std::ios::binary);
+  ASSERT_TRUE(stream_ptr->is_open());
```

Comment:
On the sg's machines I ran into the issue that the test file was never created due to a permissions error (another user has already created the test directory). The file descriptor was then never opened leading `binary_file_wrapper.cpp:get_stream` to call the mocked `filesystem_wrapper_->get_stream` more than once as every `stream_->is_open()` returned `false`. To avoid this happening I propose assertion that with a test precondition.

Reply:
I didn't really get this. How can this `ASSERT_TRUE` help with creating the test file?

Reply:
It cannot help create it, but it can prevent the tested logic from starting. That saves some debugging time as the error message in the storage component doesn't indicate the origin of this issue.